test/components/folio/console/files/show_component_test.rb

```diff
@@ -14,4 +14,123 @@ def test_render
       end
     end
   end
+
+  def test_warning_for_returns_nil_when_file_has_no_placements
+    image = create(:folio_file_image, file_placements_count: 0)
+    component = Folio::Console::Files::ShowComponent.new(file: image)
+
+    assert_nil component.warning_for(:alt)
+    assert_nil component.warning_for(:description)
+    assert_nil component.warning_for(:author)
+  end
+
+  def test_warning_for_alt_returns_nil_when_validation_is_disabled
+    Rails.application.config.stubs(:folio_files_require_alt).returns(false)
```

Comment:
Sis asi ty testy nepustila, že? Bo tohle neumíme `stubs` a `returns`. Používáme jen minitest, nikoliv Mocha.
```
    Rails.application.config.stub(:folio_files_require_alt, false) do
      ..test...
    end
```

Pokud bys chtěla i ověřit, že se ta metoda `folio_files_require_alt` vůbec volala, tak  můžeš využít náš helper
```
expect_method_called_on(object: Rails.application.config,
                            method: :folio_files_require_alt,
                            return_value: false) do
   ...test ...
end
```